packages/arb-test/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This and the rest of the packages/arb-test files should be removed. The code necessary for Fibonacci is currently just in https://github.com/OffchainLabs/demo-app-fibonacci and could be moved to arbitrum/demos if we want to. It shouldn't be left as a top level packages.